src/commands/createWebApp/setPostPromptDefaults.ts

```diff
@@ -78,12 +78,7 @@ export async function setPostPromptDefaults(wizardContext: IAppServiceWizardCont
     }
 }
 
-export async function getAppServicePlan(wizardContext: IAppServiceWizardContext, rgName: string, newPlanName: string): Promise<AppServicePlan | null> {
-    const client: WebSiteManagementClient = createAzureClient(wizardContext, WebSiteManagementClient);
-    return await client.appServicePlans.get(rgName, newPlanName);
-}
-
-function checkPlanForPerformanceDrop(asp: AppServicePlan): boolean {
+function checkPlanForPerformanceDrop(asp: WebSiteManagementModels.AppServicePlan): boolean {
```

Comment:
Just to make sure, does` client.appServicePlans.list` now also return the asp's `numberOfSites` property?

Reply:
Discussed offline